examples/huggingface/onnxruntime/speech-recognition/quantization/requirements.txt

```diff
@@ -4,7 +4,7 @@ transformers
 jiwer
 optimum
 onnx
-onnxruntime
+onnxruntime<=1.15.1
```

Comment:
only upper bound?

Reply:
it will go to the latest onnxruntime satisfy this requirement.

Reply:
I thought that if there is a onnxruntime installed on the working environment, pip will think that requirements are satisfied even if the package installed are extremely outdated if no lower bound specified.

Reply:
what's the issue not supporting latest onnxruntime?